### DISCUSSION TITLE
Add Parcelize for saving chart state in Compose

### DISCUSSION THREAD

**Discussion by BreakZero** [patrykandpatrick/vico#539] [Ideas]
`remember` block will be cleared when Composition out of screen, In some usecases we need to store state with `rememberSaveable`. And on Android, the state stored by `rememberSaveable` will save into Bundle(also SavedStateHandle of ViewModel), so need make state Parcelable.

**Comment by patrickmichalik** [comment]
Hello! Thanks for the suggestion. We’ll be looking into this.

**Comment by patrickmichalik** [comment]
[Vico 2.0.0 Alpha 9](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.9) addresses this—`rememberVicoScrollState` and `rememberVicoZoomState` use `rememberSaveable`. Cheers!

**Comment by Gowsky** [comment]
Hello. [Vico 2.0.0](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0) is now available.
